### PR TITLE
Remove outdated translations and add new translations

### PR DESCRIPTION
**Pull request by jinhojang6**
## Removed (the removed keys will show English copy on the app)

* outdated translations from all the languages
* errors in string interpolation - e.g., some languages missed variables in code like `{{something}}`

## Updated 

* new translations from contributors (https://github.com/status-im/translate.status.im#contributors)

The existing translations contain a lot of errors such as missed variables, old translations, etc. I removed all the translations with problems so that they use English copy instead.

## Files changed

* ar.json
* de.json
* es_419.json (Latin American Spanish)
* es.json
* fa.json
* fr.json
* it.json
* ja.json
* ms.json
* ne.json
* nl.json
* pl.json
* pt_BR.json (Brazil)
* pt.json
* ru.json
* th.json
* tl.json
* ko.json
* zh.json (simplified Chinese)

### CODE REVIEW

**Review by flexsurfer — APPROVED** [review]

**Review by jakubgs — APPROVED** [review]

**Comment by Serhy** [comment]
Hey @jinhojang6 
First I thought why so many translations removed... But compared to English I see they are indeed have different translations (at least in ru.json)
Well done, @jinhojang6 !

**Comment by jinhojang6** [comment]
@Serhy Thanks for testing this PR :)

File: translations/fr.json [hunk 1]
```diff
@@ -106,7 +109,7 @@
     "browsing-site-blocked-title": "Ce site est bloqué",
     "browsing-title": "Parcourir",
     "camera-access-error": "Pour accorder l'autorisation requise pour la caméra, accédez aux paramètres de votre système et assurez-vous que Status > Caméra est sélectionné.",
-    "can-not-add-yourself": "Vous ne pouvez pas vous ajouter",
+    "can-not-add-yourself": "C'est vous, pour commencer un chat, choisissez quelqu'un d'autre",
```
Comment (jrainville) [inline translations/fr.json]:
```suggestion
    "can-not-add-yourself": "C'est vous, pour commencer une conversation, choisissez quelqu'un d'autre",
```

File: translations/fr.json [hunk 2]
```diff
@@ -711,6 +726,7 @@
     "notifications": "Notifications",
     "notify": "Notifier",
     "off": "Désactivé",
+    "off-status-tree": "Arbre Status désactivé",
```
Comment (jrainville) [inline translations/fr.json]: Not 100% sure of the context, even after reading the english translation, but I don't think this is the wanted translation

**Review by jrainville — COMMENTED** [review]
Reviewed the new French translations. Two of them need changes I think

File: translations/fr.json (hunk 2 above)
Comment (guylouis) [reply translations/fr.json]: I get the context and we should translate it "En dehors de l'arbre Status", which means outside of the tree of derivation paths from the status seed :)

Comment (jrainville) [reply translations/fr.json]: Ok, that's similar to what I thought, but wanted to be sure. Thanks

Comment (jinhojang6) [reply translations/fr.json]: @jrainville @guylouis I've just updated the commit. Please take a look :)

**Review by jrainville — APPROVED** [review]
French translation looks good

**Comment by jakubgs** [comment]
@Serhy you think it's worth testing this again after #9945 was merged or can I just merge this?

Jinho rebased and it built just fine. Works fine too.

**Comment by Serhy** [comment]
@jakubgs Yes lets merge.

**Pull request merged**